app/javascript/packs/containers/OptionsContainer.jsx

```diff
@@ -38,14 +75,20 @@ class OptionsContainer extends React.Component {
       answer.id = this.props.answer.id;
       this.props.updateAnswer(answer.building_id, answer);
     }
+
+    setTimeout(() => this.focusFirstDependent(), 500);
   }
 
   render() {
```

Comment:
this function is super long--is there a way to break it up into pieces or have some helper methods take some of the logic away from here?

Reply:
`const dependentQuestions` will be moved into a component